some-service.cog.toml

```
@@ -1,44 +1,54 @@
-service_name = 'some_service'
-
-# https://toml.io/en/v1.0.0-rc.1#section-18
-# env = 'docker'
-[[docker]]
-
-[[docker.cfg]]
-[docker.cfg.var]
-value = 'var_value'
-
-[[docker.cfg]]
-[docker.cfg.other_var]
-value = 'other_var_value'
+name = ["some_service"]
```

whoops, need to revert this.
```suggestion
name = "some_service"
```